act-framework.bs

```diff
@@ -54,8 +54,55 @@ ACT Input Data {#input}
 Test Input Types {#input-types}
 -------------------------------
 
-Editor note: Some rules may run only on the template, and others require the full page to be rendered on screen before it can run. This sections describe the different types of content that can be tested by ACT Rules, and how rules should deal with each of these.
+Web pages, publications and applications go through many different stages before it is rendered to the end user. For example, PHP may be used to put various pieces of content into a template. That template is then sent as an HTML text document to a web browser, which in term parses it and turns it into a DOM tree, before rendering it to the screen. At each of these stages, accessibility tests could be run. We can divide this into two stages of testing, black-box and white-box testing.
```

Comment:
+1 to Shadi's comment. 
Some additional comments:

- Web pages are plural. "Web pages, including publications and applications, go through many different stages before **_they are_** rendered to the end user."

- At each of these stages, accessibility tests **_can_** be run. We can divide this into two stages of testing, black-box**_,_** and white-box testing.

- I think we need to provide some definitions here for DOM, black-box and white-box testing.


-